### DISCUSSION TITLE
home readme url 在首页不显示。另显示We're sorry but alist doesn't work properly without JavaScript enabled. Please enable it to continue.

### DISCUSSION THREAD

**Discussion by sinkers-lan** [AlistGo/alist#802] [Q&A]
![image](https://user-images.githubusercontent.com/100743317/159500297-9309bc52-a418-437c-8abb-029af0ce8f66.png)
![image](https://user-images.githubusercontent.com/100743317/159500590-e4a4eeef-c667-4bbb-8909-f5bbc6e8feb1.png)
home readme url 在首页不显示，另显示We're sorry but alist doesn't work properly without JavaScript enabled. Please enable it to continue.
请问这是什么意思？是我readme url填的格式不对，还是有什么东西没装呢？我用的是一键安装的脚本`curl -fsSL "https://nn.ci/alist.sh" | bash -s install`。

**Comment by xhofe** [comment]
https://developer.mozilla.org/zh-CN/docs/Learn/Common_questions/What_is_a_URL

**Reply by sinkers-lan** [reply]
![image](https://user-images.githubusercontent.com/100743317/159659231-82e9bb98-dc95-432d-b92d-dddaa1588f71.png)
![image](https://user-images.githubusercontent.com/100743317/159658662-cd0a6022-2757-4d8c-a11c-42837f9a8331.png)
我之前填的没带`https://`。我如果填了`https://我的域名/README.md`，首页里就会一直转圈，一直加载不出来就像上图这样

**Reply by xhofe** [reply]
请确认你的链接可访问且支持cors。
